UnityPy/helpers/MeshHelper.py

```diff
@@ -650,41 +654,31 @@ def get_triangles(self) -> List[List[Tuple[int, ...]]]:
             indexCount = m_SubMesh.indexCount
             topology = m_SubMesh.topology
 
-            triangles: List[Tuple[int, ...]]
+            triangles: List[Tuple[int, ...]] = []
 
             if topology == MeshTopology.Triangles:
-                triangles = self.m_IndexBuffer[firstIndex : firstIndex + indexCount]  # type: ignore
-                triangles = [triangles[i : i + 3] for i in range(0, len(triangles), 3)]  # type: ignore
-            elif self.version[0] < 4 or topology == MeshTopology.TriangleStrip:  # TriangleStrip
-                # todo: use as_strided, then fix winding, finally remove degenerates
-                triIndex = 0
-                triangles = [None] * (indexCount - 2)  # type: ignore
-
-                for i in range(indexCount - 2):
-                    a, b, c = self.m_IndexBuffer[firstIndex + i : firstIndex + i + 3]
+                for i in range(firstIndex, firstIndex + indexCount, 3):
```

Comment:
Python for loops are way slower than list comprehension.
While it's tricky to use it for the other versions,
it's rather easy for normal triangles to use, and should therefore be used.